api/src/main/java/org/openmrs/module/eptsreports/reporting/library/datasets/BaseDataSet.java

```diff
@@ -36,17 +35,20 @@ protected void addRow(CohortIndicatorDataSetDefinition cohortDsd, String baseNam
 		
 		private String dimensions;
 		
+		private String column;
+		
 		/**
 		 * Default constructor
 		 * 
 		 * @param name the name
 		 * @param label the label
 		 * @param dimensions the dimension parameters
 		 */
-		public ColumnParameters(String name, String label, String dimensions) {
+		public ColumnParameters(String name, String label, String dimensions, String column) {
 			this.name = name;
```

Comment:
Do we still need the "name" property?

Reply:
That is how I would Identify my column uniquely if I would need to reference by name